src/css/header.css

```diff
@@ -180,7 +180,7 @@ body {
   color: var(--navbar-menu-font-color);
   display: block;
   line-height: var(--doc-line-height);
-  padding: 0.5rem 1rem;
+  padding: 0.5rem 1.5rem;
```

Comment:
This also increases padding around all the drop-down menus, Get Started and search icon in the header.